specification/operationalinsights/data-plane/readme.nodejs.md

```diff
@@ -0,0 +1,15 @@
+
+## NodeJS
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to the root directory of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  package-name: azure-loganalytics
+  package-version: 0.1.0
+  output-folder: $(node-sdks-folder)/lib/services/loganalytics
```

Comment:
The management plane package is named as `azure-arm-operationalinsights`. It can be found here https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/operationalInsightsManagement/package.json.

Is there a reason to not follow the convention to name it as `azure-operationalinsights` and name it as `azure-loganalytics` ?

Reply:
Operational Insights is old branding. In general I am trying not to produce fresh releases that use old branding, and will migrate older code to the new branding as well. The RP is still microsoft.operationaLinsights.

Reply:
See e.g. Python management cmdlets for same service: https://github.com/Azure/azure-sdk-for-python/tree/master/azure-mgmt-loganalytics

Reply:
sounds good to me.